R/radsl.R

```diff
@@ -273,7 +273,7 @@ radsl <- function(N = 400, # nolint
   # apply metadata
   if (is.null(with_trt02)) { # nolint
     ADSL <- apply_metadata(ADSL, "metadata/ADSL.yml", FALSE) # nolint
-  } else {
+  }else{# nolint
```

Comment:
`# nolint` should be used only for exceptions in naming conventions such as `ADSL` (we use `lower_snake_case`). Here, it should be `} else {`. Other standards like these can be found in the [dev-docs](https://nest.pages.roche.com/docs/dev-guide/coding/).

Reply:
Thanks @Melkiades The code format is updated